vitrina/datasets/models.py

```diff
@@ -153,6 +153,16 @@ def get_absolute_url(self):
     def get_tag_list(self):
         return list(self.tags.all().values_list('name', flat=True))
 
+    @property
+    def parent_category(self):
+        categories = []
+        if self.category:
+            category = self.category
+            while category.parent:
+                category = category.parent
+            categories.append(category.pk)
+        return categories
```

Comment:
One of the reasons why we use treebeard, is to avoid expensive code like this one, involving multiple queries. Please use a treebeard method instead.